modules/ROOT/pages/introduction.adoc

```diff
@@ -62,8 +54,8 @@ The highlighted components in each phase emphasize how your client applications
 
 ==== Pre-migration client application operations
 
-Let's look at a pre-migration from a high-level view.
-At this point, your client applications are performing read/write operations with an existing CQL-compatible database: Apache Cassandra, DSE, or Astra DB.
+Here's a look at a pre-migration from a high-level view.
+At this point, your client applications are performing read/write operations with an existing CQL-compatible database: Apache Cassandra, DSE, or {astra_db}.
```

Comment:
I'd suggest changing the `:` to `such as`. This is a nuance, but we want to imply that there are other CQL-compatible databases that you may want to move from without mentioning them explicitly.